index.html

```diff
@@ -8,26 +8,102 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="Shopping Cart Page">
     <meta name="author" content="Rachel">
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
-    <script src="js/application.js"></script>
-    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
     <link href="css/custom.css" rel="stylesheet">
   </head>
-
   <body>
 <!-- Landing Page -->
     <h1>Urban Fictionary</h1>
-      <div>
-        <p>Player 1</p>
-        <p>Player 2</p>
-        <section>
-          Description: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-        </section>
-       </div>
-       
+      <section id="page">
+        <p class="introduction">
+          That 
+        </p>
+        <form>
+          <label>Player 1: <input id="1stplayer" placeholder="Name" /></label>  
+          <label>Player 2: <input id="2ndplayer" placeholder="Name" /></label>   
+        </form>
+        <button type="button" class="btn startingButton">Start the Game!</button>
+       </section>   
+
 <!-- Game Page  -->
-  
+    <div class="container" id="page2">
+      <div class="row">
+  <!-- Player Details     -->
+        <div class="col-xs-3">
+          <aside class="player-names">
+            <h3>PLAYERS</h3>
+            <ul>
+            </ul>
+          </aside>
+        </div>
+  <!-- Main Game       -->
+        <div class="col-xs-6 accordion" id="q1">
+          <main>
+            <h2> QUESTION 1 </h2>
+              <div class="pane">
+                <span class="word">Just Ducky</span>
+                <ul>
+                  <li class="choice">A) Calm on the surface, paddling like crazy beneath.</li>
+                  <li class="choice">B) Just fine and dandy - like Donald.</li>
+                  <li class="choice">C) Pleasing and cute.</li>
+                </ul>
+              </div>
+            <h2> QUESTION 2 </h2>
+              <div class="pane">          
+              </div>
+            <h2> QUESTION 3 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 4 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 5 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 6 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 7 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 8 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 9 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 10 </h2>
+              <div class="pane">
+              </div>
+            <h2> QUESTION 11 </h2>
```

Comment:
Instead of hard coding these questions in HTML, you can dynamically generate them in JavaScript